WinUIGallery/Controls/PageHeader.xaml

```diff
@@ -0,0 +1,165 @@
+        <TextBlock
+            x:Name="NamespaceTextBlock"
+            Grid.Row="1"
+            Margin="0,4,0,0"
+            Foreground="{ThemeResource TextFillColorSecondaryBrush}"
+            IsTextSelectionEnabled="True"
+            Style="{StaticResource CaptionTextBlockStyle}"
+            Text="{x:Bind Item.ApiNamespace, Mode=OneWay}"
+            Visibility="{x:Bind Item.ApiNamespace, Converter={StaticResource emptyStringToVisibilityConverter}, Mode=OneWay}" />
+        <Grid Grid.Row="2" Margin="0,12,0,12">
+            <StackPanel Orientation="Horizontal" Spacing="4">
+                <DropDownButton
+                    AutomationProperties.Name="Documentation"
+                    ToolTipService.ToolTip="Documentation"
+                    Visibility="{x:Bind Item.Docs, Converter={StaticResource collectionConverter}}">
+                    <DropDownButton.Content>
+                        <StackPanel Orientation="Horizontal" Spacing="8">
+                            <FontIcon FontSize="14" Glyph="&#xE130;" />
+                            <TextBlock Text="Documentation" />
+                        </StackPanel>
+                    </DropDownButton.Content>
+                    <DropDownButton.Flyout>
+                        <Flyout Placement="Bottom">
+                            <ItemsControl
+                                x:Name="DocsList"
+                                IsTabStop="False"
+                                ItemsSource="{x:Bind Item.Docs}">
+                                <ItemsControl.ItemsPanel>
+                                    <ItemsPanelTemplate>
+                                        <StackPanel Orientation="Vertical" />
+                                    </ItemsPanelTemplate>
+                                </ItemsControl.ItemsPanel>
+                                <ItemsControl.ItemTemplate>
+                                    <DataTemplate x:DataType="data:ControlInfoDocLink">
+                                        <HyperlinkButton NavigateUri="{Binding Uri}" ToolTipService.ToolTip="{Binding Uri}">
+                                            <TextBlock Text="{x:Bind Title}" />
+                                        </HyperlinkButton>
+                                    </DataTemplate>
+                                </ItemsControl.ItemTemplate>
+                            </ItemsControl>
+                        </Flyout>
+                    </DropDownButton.Flyout>
+                </DropDownButton>
+                <DropDownButton AutomationProperties.Name="Source code" ToolTipService.ToolTip="Source code">
+                    <DropDownButton.Content>
+                        <StackPanel Orientation="Horizontal" Spacing="8">
+                            <PathIcon
+                                Margin="-3,-3,-16,-3"
+                                VerticalAlignment="Center"
+                                Data="{StaticResource GitHubIconPath}">
```

Comment:
Is there really no better way to do this?

Reply:
This was something I had laying around for the Toolkit Gallery :).. 

SVG is probably best? We do want to set the Foreground so it renders correctly in light/dark theme

Reply:
I think we can do the fill stuff with an SVG too. I also still got https://github.com/CommunityToolkit/WindowsCommunityToolkit/pull/4494/files open which touches on the topic of it not being properly centered in the WCT Gallery.